FindReferenceLAPACK.cmake

```diff
@@ -46,6 +46,12 @@ else()
   set( ReferenceLAPACK_LIBRARIES ${ReferenceLAPACK_LP64_LIBRARIES} )
 endif()
 
+find_package(StandardFortran REQUIRED)
```

Comment:
This can't be here, I have a module that checks for these things

https://github.com/wavefunction91/linalg-cmake-modules/blob/main/util/BLASUtilities.cmake#L43

Though I would like to see this `StandardFortran` module get used there